lib/index.js

```diff
@@ -8,8 +8,10 @@ let { abort, repr } = require("./util");
 let SerializedRunner = require("./util/runner");
 let browserslist = require("browserslist");
 
-module.exports = (referenceDir, config,
-		{ watch, fingerprint, sourcemaps, compact, serve, liveserve }) => {
+module.exports = async function faucetDispatch(referenceDir, config,
+		{ watch, fingerprint, sourcemaps, compact, serve, liveserve }) {
+	config = await config;
```

Comment:
Our very first use of `await` could hardly have been more handy (cf. commit message).

Reply:
Yep, totally. I think this is a good idea.